float64[24] jacobian